src/odemis/driver/hamamatsurx.py

```diff
@@ -1124,8 +1166,7 @@ def readCommandResponse(self):
                         try:
                             additional_info = additional_info.split("\r")[:-1]
                             for i, item in enumerate(additional_info):
-                                rargs.append(item.replace("\n", ""))  # TODO need str instead of list
-                                # rargs[0] = rargs[0] + ", " + item.replace("\n", "")  # TODO something like this test on HW!!
+                                rargs.append(item.replace("\n", ""))
```

Comment:
Are there `\n` anywhere in the string, or just at the end? If so, a .strip() would be clearer and cleaner.

Reply:
I think there was the problem that there were \n within the string. I will put a TODO and check on the HW again.